plugins/search/src/components/SearchResult/SearchResult.tsx

```diff
@@ -159,8 +166,25 @@ export const SearchResult = ({ searchQuery }: SearchResultProps) => {
       setFilteredResults(withFilters);
     }
   }, [filters, searchQuery, results]);
-
-  if (loading || error || !results) return null;
+  if (loading) {
+    return <Progress />;
+  }
+  if (error) {
+    return (
+      <Alert severity="error">
+        Error encountered while fetching search results. {error.toString()}
+      </Alert>
+    );
+  }
+  if (!results || results.length === 0) {
+    return (
+      <EmptyState
+        missing="info"
+        title="No information to display"
+        description="Sorry, no results were found"
```

Comment:
Wohoo this is great!! 🎉  Just one comment - can we switch the title to be "Sorry, no results were found" and take away the description prop.

Reply:
and also change to `missing="data"`